### DISCUSSION TITLE
running from dhclient-exit-hooks

### DISCUSSION THREAD

**Discussion by RanceH** [troglobit/inadyn#376] [Ideas]
I have plans for a FreeBSD 13 box to serve as a home router/gateway.  My public IP is assigned to my wan interface, so I'm aware of IP changes through dhclient.

Instead of running inadyn as a service, I'm wondering about the possibility of "runonce" support so that it can run from dhclient-exit-hooks.  I already have an exit-hooks script so its trivial to add inadyn to that script and run it once when its needed instead of as a service on a timer. The FreeBSD init script doesn't suggest that such a thing is possible.

**Comment by RanceH** [comment]
Never mind.  The man page says that -once is an option, so with that, I apologize for the noise and I'll figure it out.

**Reply by troglobit** [reply]
Ah great, you found it yourself!  No apologies necessary, good luck :smiley:
